source/docs/software/vision-processing/introduction/game-piece-detection.rst

```diff
@@ -0,0 +1,227 @@
+
+### Calculating Angle to Target
+
+Once you know your FOV and the target's position in normalized coordinates, you can calculate the angle from your camera to the target:
+
+.. math::
+   \textit{pitch}=A_y \cdot \frac{1}{2}FOV_{vertical}
+
+.. math::
+   \textit{yaw}=A_x \cdot \frac{1}{2}FOV_{horizontal}
+
+These angles tell you how far up/down and left/right the game piece is relative to your camera's center axis.
+
+### Calculating Distance to Target
+
+If your camera is mounted at a known angle and height, and you know the approximate height of the game piece, you can calculate the distance:
+
+.. math::
+   \textit{distance}=\frac{\textit{height}_{target}-\textit{height}_{camera}}{\tan(\textit{angle}_{camera}+\textit{pitch})}
+
+Where:
+
+- **height_target**: The height of the game piece (often at or near ground level)
+- **height_camera**: The mounting height of your camera
+- **angle_camera**: The angle your camera is tilted (positive for tilted down, negative for tilted up)
+- **pitch**: The calculated pitch angle to the target
+
+.. important:: The camera mounting angle is critical for accurate distance measurements. A camera pointed straight ahead cannot accurately determine distance to objects at the same height. Angling the camera down 15-30 degrees typically provides good results for ground-level game pieces.
+
+### Estimating Target Pose
+
+For more advanced applications, you may want to estimate the full 3D pose (position and orientation) of the game piece relative to your robot. This requires:
+
+1. **Known dimensions** of the game piece
+2. **Calibrated camera** to account for lens distortion
+3. **Multiple features** on the object to determine orientation
+
+For circular game pieces, orientation may not matter. For asymmetric objects, detecting specific features (corners, markings, etc.) can help determine how the object is oriented.
+
+The WPILib pose estimation classes can incorporate vision measurements to help track your robot's position and plan paths to game pieces.
+
+### Alternative: Area-to-Distance Lookup
+
+If your game piece is always oriented the same way, you can create a lookup table correlating particle area (in pixels) to distance. This is less accurate than trigonometric methods but can be simpler to implement:
+
+1. Place the game piece at known distances from your camera
+2. Measure the particle area at each distance
+3. Create a lookup table or fit a curve to the data
+
+Note that area varies with the inverse square of distance, so small measurement errors at large distances can cause significant inaccuracy.
+
+## Choosing a Camera
+
+Selecting the right camera is crucial for reliable game piece detection. Key considerations include:
+
+### Field of View (FOV)
+
+- **Wide FOV (>100°)**: Better for detecting game pieces at close range and wide areas, but objects appear smaller at distance
```

Comment:
I'd mention something about distortion for wide FOV cameras that may decrease accuracy